### PR TITLE
feat(app): add virumap VIR-427

### PR DESCRIPTION
**Pull request by ferlrodriguez**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,107 @@
+{
+  "name": "virumap-app",
+  "version": "0.0.17",
```
Comment (mduthey) [inline package.json]: Should start from 0.0.1

File: src/containers/Welcome/Step1/Step1.tsx [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+
+  useEffect(() => {
+    i18n.changeLanguage(lang);
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [i18n, lang]);
+
+  /* This `useEffect` hook is responsible for initializing the form values for the country and language
+  fields when the component mounts. Here's a breakdown of what it does: */
+  useEffect(() => {
+    const localStorageCountry = localStorage.getItem('countryResult');
```
Comment (mduthey) [inline src/containers/Welcome/Step1/Step1.tsx]: Change this `countryResult` to `virumap-countryResult` also. If you used it previously, maybe you have more languages than are really supported

**Review by mduthey — APPROVED** [review]
check the 2 requests

File: src/containers/Welcome/Step1/Step1.tsx (hunk 2 above)
Comment (ferlrodriguez) [reply src/containers/Welcome/Step1/Step1.tsx]: https://github.com/virufy/virumap-app/pull/1/commits/27f2aa00ef60acba2f2f14ec4efe9197fa1ff51b

File: package.json (hunk 1 above)
Comment (ferlrodriguez) [reply package.json]: https://github.com/virufy/virumap-app/pull/1/commits/27f2aa00ef60acba2f2f14ec4efe9197fa1ff51b

**Pull request merged**